source/sync/configure/sync-settings.txt

```diff
@@ -173,11 +201,64 @@ Performance and Storage
 Each queryable field adds additional metadata storage to your Atlas cluster and
 may lead to degraded write performance. You should have as few queryable fields
 as needed by your application. A good rule of thumb is to have at most 10
-queryable fields.
+queryable fields. If you need to reduce storage usage or improve performance, 
```

Comment:
I think this rule of thumb can be updated to something like "10 queryable fields applying to any single collection" to indicate that between an apps global+collection queryable fields they should aim for less than 10 to apply to any collection